css/style.css

```diff
@@ -0,0 +1,493 @@
+body{
+    font-size: 0.8rem;
+    font-family: 'Roboto', Arial, sans-serif;
+}
+
+hr {
+    border: 1px solid #e3e3e3;
+}
+#nav-bar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    height: 36px;
+    margin: 20px;
+    }
+    
+    #col1{
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-start;
+        
+    }
+
+    #col1 #logo{
+        width: 80px;
+        height: 24px;
+        margin-top: 10px;
+    }
+    
+    #col2{
+        display: flex;
+        flex-direction: row;
+        flex: 0.7;
+        margin-top: 10px;
+        justify-content:center;
+
+    }
+
+    #col2  form{
+        display: flex;
+        flex:1;
+        height: 32px;
+    }
```

Comment:
- [ ] You are repeating the same rules in different parts of your CSS (like `display: flex;` or `color:#666464;`). For good coding practices ([check this link](https://speckyboy.com/good-bad-css-practices)), follow the DRY rule (Don't repeat yourself).
![image](https://user-images.githubusercontent.com/53256139/89356769-33203f80-d684-11ea-8a2b-7776e5a73811.png)


  👉 Here you have an explanation of how you can implement it:

  You can use multiple classes for any HTML element, so for example,

  If you use these lines in many classes:
  ```
  color: #fff;
  background-color: #000;
  ```
  and you use it for .class1, .class2, and .class3 
  you can put them inside a single class called for example "common-class" so it will look like this:
  **CSS**
  ```
  .class1 {
    font-size: 24px;
  }
  .class2 {
    font-size: 30px;
  }
  .class3 {
    font-size: 10px;
  }
  .common-class {
    color: #fff;
    background-color: #000
  }
  ```
  and in **HTML**:
  ```
  <nav class="class1 common-class">  <!-- this will have font size of 24px, color of #fff and background-color of #000 -->
  ...
  <div class="class2 common-class">  <!-- this will have font size of 30px, color of #fff and background-color of #000 -->
  ...
  <footer class="class3 common-class">   <!-- this will have font size of 10px, color of #fff and background-color of #000 -->
  ...
  ```